Bottom: 89ebfd60338e60917ebaef4e94b7d948d84176dd
Top:    6a757a10982c7c5d0f1a3ce7624a449699e59106
Author: Jason Wang <jasowang@redhat.com>
Date:   2015-05-11 16:05:45 +0800

Refresh of vhost-let-vhost_signal-returns

---

diff --git a/drivers/vhost/vhost.h b/drivers/vhost/vhost.h
index faa0dd7..e0ec89c 100644
--- a/drivers/vhost/vhost.h
+++ b/drivers/vhost/vhost.h
@@ -166,6 +166,8 @@ void vhost_add_used_and_signal(struct vhost_dev *, struct vhost_virtqueue *,
 void vhost_add_used_and_signal_n(struct vhost_dev *, struct vhost_virtqueue *,
 			       struct vring_used_elem *heads, unsigned count);
 void vhost_signal(struct vhost_dev *, struct vhost_virtqueue *);
+void vhost_check_coalesce_and_signal(struct vhost_dev *dev,
+				     struct vhost_virtqueue *vq);
 void vhost_disable_notify(struct vhost_dev *, struct vhost_virtqueue *);
 bool vhost_enable_notify(struct vhost_dev *, struct vhost_virtqueue *);
